dashboard/app/models/course.rb

```diff
@@ -51,7 +51,7 @@ def self.load_from_path(path)
     hash = JSON.parse(serialization)
     course = Course.find_or_create_by!(name: hash['name'])
     course.update_scripts(hash['script_names'])
-    course.update!(properties: hash['properties'])
+    course.update!(teacher_resources: hash.try(:[], 'properties').try(:[], 'teacher_resources'))
```

Comment:
Maybe I'm simply lacking context...but I'm confused. This seems like either (a) we need to special case `teacher_resources` or (b) we should be doing `course.properties = {}` before loading from `serialization`. But I don't understand how there should be any brittleness for future properties.